src/utils/Utilities.java

```diff
@@ -0,0 +1,50 @@
+package utils;
+
+import data.Card;
+import data.Color;
+import data.Language;
+
+import java.util.*;
+
+public class Utilities {
+
+    public static Optional<String> getLanguageFromUser() {
```

Comment:
skoro stworzyles enuma to zwroc go `Optional<Language> getLanguageFromUser()` i uzywaj w reszczie kodu.